src/deepsparse/loggers/helpers.py

```diff
@@ -34,11 +34,60 @@
     "NO_MATCH",
     "access_nested_value",
     "finalize_identifier",
+    "unwrap_logs_dictionary",
 ]
 
 NO_MATCH = "NO_MATCH"
 
 
+def unwrap_logs_dictionary(
+    value: Any, parent_identifier="", seperator="__"
```

Comment:
let's default separator to `/` since this is what we use in actual pipeline logging. the `__` is prometheus specific and will be injected later, correct?

Reply:
Not really.
To reiterate:

Identifiers in our logging framework have three kinds of separators: `/`, `.`, and `__`.
The convention is:
`whatever/we/want/to/have/here.remainder__metric_function_name__subname__subsubname`
e.g.
`image_classification/pipeline_inputs.images__mean_pixels_per_channel__0`

So this identifier is not particular to Prometheus formatting. It continues to append nested subnames to the output of the metric function.